lightworker/__init__.py

```
@@ -1,3 +1,5 @@
 """LightWorker: a small, local-first multi-agent task runner."""
 
+# CodeRabbit 配置验证测试：此改动用于验证自动代码评审是否正常触发
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)